views/fanChart/FanChartView.js

```diff
@@ -1140,7 +1165,18 @@
         specLocSelectorBR.style.display = "none";
         specFamSelectorBR.style.display = "none";
 
-        // console.log(theCheckIn);
+        // SOME minor tweaking needed in the HIGHLIGHT tab of the Settings object since some drop-downs are contingent upon which original option was chosen
+        let highlightSelector = document.getElementById("highlight_options_highlightBy");
+        highlightSelector.setAttribute("onchange", "FanChartView.optionElementJustChanged();");
+        let break4DNASelector = document.getElementById("highlight_options_break4DNA");
+        let howDNAlinksSelector = document.getElementById("highlight_options_howDNAlinks");
+        let catNameSelector = document.getElementById("highlight_options_catName");
+        let catNameSelectorLabel = document.getElementById("highlight_options_catName_label");
+        catNameSelector.style.display = "none";
+        catNameSelectorLabel.style.display = "none";
+        
+
+        console.log("TWEAKED the Highlights tab - how many categories I wonder ...", categoryList);
```

Comment:
There are lots of these (debugging) console logs throughout your code. Would it not be better to make use of the `condLog` available in tree.js? (Just remember to turn `debugLoggingOn` on in index.html, and off again before pushing a PR) Then people's browser memory will also be slightly less reduced by running your apps :)